src/transport/SecureSessionMgr.cpp

```diff
@@ -250,9 +253,9 @@ CHIP_ERROR SecureSessionMgr::NewPairing(const Optional<Transport::PeerAddress> &
             state, [this](const Transport::PeerConnectionState & state1) { HandleConnectionExpired(state1); });
     }
 
-    ChipLogDetail(Inet, "New pairing for device %llu, key %d!!", peerNodeId, peerKeyId);
-
-    state = nullptr;
+    ChipLogDetail(Inet, "New pairing for device 0x%08lx%08lx, key %d!!", static_cast<unsigned long>(peerNodeId >> 32),
+                  static_cast<unsigned long>(peerNodeId), peerKeyId),
```

Comment:
```suggestion
                  static_cast<unsigned long>(peerNodeId), peerKeyId);
```

Reply:
> 

good catch - corrected.
Didn't give a compiler error as it's still valid C/C++ - nasty one.